packages/abi/scripts/generate-abi.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+## Build the contracts
+yarn build:contracts
```

Comment:
I ran this locally, it reverts in one of the scripts if the ALCHEMY KEY is not set in the `.env` but then continues anyway. It should probably revert and exit. Same thing for requiring a private key to be set. I think that's because you put build contracts here instead of in the package.json so:

"generate:abi": "yarn build:contracts && ./script"

Would terminate the script if you have reverts in the prior stages. It's also a bit easier to see what's happening.